pkg/scheduler/plugins/gang/gang.go

```diff
@@ -83,15 +83,26 @@ func (gp *gangPlugin) OnSessionOpen(ssn *framework.Session) {
 	preemptableFn := func(preemptor *api.TaskInfo, preemptees []*api.TaskInfo) ([]*api.TaskInfo, int) {
 		var victims []*api.TaskInfo
 		jobOccupiedMap := map[api.JobID]int32{}
-
+		jobAllocatedMap := map[api.JobID]*api.Resource{}
 		for _, preemptee := range preemptees {
 			job := ssn.Jobs[preemptee.Job]
 			if _, found := jobOccupiedMap[job.UID]; !found {
 				jobOccupiedMap[job.UID] = job.ReadyTaskNum()
+				jobAllocatedMap[job.UID] = util.GetAllocatedResource(job)
+			}
+
+			// It's not allowed to reclaim or preempt task when allocated is less equal to min resource of job.
+			jobMinResource := job.GetMinResources()
+			klog.V(4).Infof("Job name is <%v>, Allocated is %+v, MinResource is %+v.", job.Name, jobAllocatedMap[job.UID], jobMinResource)
+			if jobAllocatedMap[job.UID].LessEqualPartly(jobMinResource, api.Zero) {
```

Comment:
Before judging with jobMinAvailble, is it more reasonable to subtract the resource information of preemptee from the allocated resources?